src/pages/index.js

```diff
@@ -36,82 +82,213 @@ const editCardPopup = new PopupWithForm(
 );
 editCardPopup.setEventListeners();
 
-const userInfo = new UserInfo({
-  nameSelector: ".profile__title",
-  jobSelector: ".profile__description",
-});
-
-const cardSection = new Section(
+const editAvatarPopup = new PopupWithForm(
   {
-    items: initialCards,
-    renderer: (cardData) => {
-      const cardElement = createCard(cardData); // Create card using createCard
-      cardSection.addItem(cardElement); // Add the card to the DOM
-    },
+    popupSelector: "#avatar-edit-modal",
   },
-  ".cards__list" // Container selector
+  handleEditAvatarSubmit
 );
+editAvatarPopup.setEventListeners();
 
 const popupWithImage = new PopupWithImage({
   popupSelector: "#previewModal",
 });
 popupWithImage.setEventListeners();
 
+const confirmPopup = new ConfirmPopup({
+  popupSelector: "#delete-modal",
+});
+confirmPopup.setEventListeners();
+
+// Instantiate form validators
 const editFormValidator = new FormValidator(settings, profileEditForm);
 const addFormValidator = new FormValidator(settings, addCardForm);
+editFormValidator.enableValidation();
+addFormValidator.enableValidation();
+
+// Helper Functions
+function createCard({ name, link, _id, isLiked }) {
+  // Instantiate the card
+  const card = new Card(
+    { name, link, _id, isLiked },
+    "#card-template",
+    handleImageClick,
+    handleLikeClick,
+    handleUnlikeClick,
+    handleDeleteCard
+  );
+
+  const cardElement = card.getView();
+  const cardContainer = document.querySelector(".modal__container");
+
+  cardContainer.appendChild(cardElement);
 
-// Render initial cards
-cardSection.renderItems();
+  const deleteButton = cardElement.querySelector(".card__delete-button");
 
-// Functions
+  if (deleteButton) {
+    deleteButton.addEventListener("click", () => {
+      confirmPopup.open();
+    });
+  } else {
+    console.error("Delete button not found within card template.");
+  }
 
-// Function to create a new card
-function createCard(cardData) {
-  const card = new Card(cardData, "#card-template", handleImageClick); // Instantiate the new Card
-  const cardElement = card.getView(); // Get the card's HTML element
-  return cardElement; // Return the card's HTML element
+  return cardElement;
 }
 
-/* Event Handlers */
+//Temporary function to upload cards to server, if you want me to delete it
+// I can
 
-function handleProfileEditSubmit(formData) {
-  userInfo.setUserInfo({
-    name: formData.title,
-    description: formData.description,
+let cardUploaded = true;
+
+function uploadCardsToServer() {
+  if (cardUploaded) {
+    console.log("Cards already uploaded to server.");
+    return;
+  }
+
+  cards.forEach((card) => {
+    api
+      .createCard(card)
+      .then((addedCard) => {
+        console.log("Card added to server", addedCard);
+      })
+      .catch((err) => {
+        console.error("Error adding card to server", err);
+      });
```

Comment:
Please, delete it


You render the cards in line 68 after getting them from the server